modules/Documents/Document.php

```diff
@@ -383,6 +383,8 @@ function bean_implements($interface) {
 		switch ($interface) {
 			case 'ACL' :
 				return true;
+            case 'FILE' :
```

Comment:
can you please, fix the tab indents in this file?

Reply:
?
![screenshot_20180109_165704](https://user-images.githubusercontent.com/12231216/34732814-3f07566c-f55e-11e7-9f1c-2a0a97342d15.png)

Reply:
![image](https://user-images.githubusercontent.com/13831178/34733519-932aba8e-f560-11e7-8d00-01d15c368886.png)

Reply:
1 tab = 4 spaces

Reply:
http://www.php-fig.org/psr/psr-2/
"Code MUST use 4 spaces for indenting, not tabs."

Reply:
My IDE has let me down :( . I have re formatted these files